docs/demo_05.md

```diff
@@ -0,0 +1,27 @@
+# Load testing the child chain and Watcher
+
+The following demo is a mix of commands executed in IEx (Elixir's) REPL (see [manual startup](/docs/manual_service_startup.md) for instructions) and shell.
+
+Follow the [docs/demo_03.md](/docs/demo_03.md) to run a developer's Child chain server, Watcher and fill a testnet with noticable amount of transactions with running perftest.
+
+In the elixir REPL continue with
+```elixir
+
+### PREPARATIONS
+
+alias OMG.Performance.ByzantineEvents
+alias OMG.Performance.ByzantineEvents.Generators
+ 
+dos_users = 5
```

Comment:
I'm missing what "dos" means in this context. I think those are just `users` and `exits_per_user`, no?

Reply:
Little terminology clashing - I'll fix
Here we have spenders and `dos_users` describe how many threads will fetch exit data

Reply:
this?